ui-guidelines/css/guidelines.css

```diff
@@ -165,8 +165,60 @@ pre {
 	font-size: 1em;
 	font-weight: bold;
 }
+
+/*==== TYPOGRAPHY ====*/
+
+.font-weights {
+	width: 100%;
+	height: 10em;
+}
+.weight-sample {
+	width: 20%;
+	float: left;
+	display: block;
```

Comment:
You can omit it. When you use `float` property, then eveything becomes `display: block;`